tests/subsys/net/lib/aws_fota/aws_fota_json/src/main.c

```diff
@@ -20,16 +20,18 @@ static void test_notify_next(void)
 	char job_id[100];
 	char hostname[100];
 	char file_path[1000];
+	int version_number;
 
 	/* Memset to ensure correct null-termination */
 	memset(job_id, 0xff, sizeof(job_id));
 	memset(hostname, 0xff, sizeof(hostname));
 	memset(file_path, 0xff, sizeof(file_path));
 
-	ret = aws_fota_parse_notify_next_document(encoded, sizeof(encoded) - 1,
-			job_id, hostname, file_path);
+	ret = aws_fota_parse_job_execution(encoded, sizeof(encoded) - 1,
+			job_id, hostname, file_path, &version_number);
 
 	zassert_true(!strcmp(job_id, expected_job_id), NULL);
+	zassert_true(version_number == 1, NULL);
```

Comment:
If you use `zassert_equal` the error message will print the actual and expected value.

Reply:
Added